### PR TITLE
Update jacoco, javadoc and codeowners for monitor query

### PR DESCRIPTION
**Pull request by srnagar**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -84,6 +84,10 @@
         <!-- Models Repository -->
         <module>sdk/modelsrepository/azure-iot-modelsrepository</module>
 
+        <!-- Azure Monitor -->
+        <module>sdk/monitor/azure-monitor-opentelemetry-exporter</module>
```
Comment (pallavit) [inline pom.xml:88]: What is this for?

**Review by pallavit — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (srnagar) [reply pom.xml:88]: This is for generating an aggregate javadoc that includes all client libraries in a single javadoc [index.html](https://azuresdkartifacts.blob.core.windows.net/azure-sdk-for-java/staging/apidocs/index.html)

**Pull request merged**